backstop/tests/alert.html

```diff
@@ -3,13 +3,30 @@
   See LICENSE.md in the project root for license terms and full copyright notice.
 -->
 <!DOCTYPE html>
-<html id="theme">
+<html
+  lang="en-US"
+  id="theme"
+>
   <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
     <title>Alert | iTwinUI</title>
     <link
       rel="icon"
       href="../assets/favicon.ico"
```

Comment:
so this one is not needed anymore. we can remove links an the icon itself

Reply:
so.. ico did not work on all devices? or why you decided to change it to svg and add png for apple?

Reply:
ICO favicon does work on all devices.  SVG favicon [works on modern browsers](https://caniuse.com/link-icon-svg) but also allows us to change the favicon colors to match the OS theme.
![Screen Shot 2021-07-16 at 2 04 25 PM](https://user-images.githubusercontent.com/849817/125990428-b386bcc7-c987-4d7a-860d-90edba50a90a.png)
![Screen Shot 2021-07-16 at 2 04 42 PM](https://user-images.githubusercontent.com/849817/125990432-0578f5ee-dadf-4791-8bd5-829c9fae86c6.png)

The `apple-touch-icon` is actually not used for favicons at all, it is used if someone pins the site to their home screen.
![IMG_08BD44954855-1](https://user-images.githubusercontent.com/849817/125991264-0367929f-8ede-45d1-8863-0daf0d2bedd8.jpeg)

I listed the ICO favicon first & then the SVG favicon second thinking that if the browser supports SVG favicons, it would override the ICO favicon.  However, it appears that all of my devices are showing the ICO favicon instead of the SVG version.  I might just remove the ICO all together.

Reply:
i think you can remove the apple touch icon too. it should use the favicon as fallback

Reply:
If you remove the apple touch icon it looks like this:
![IMG_08D62215CC6B-1](https://user-images.githubusercontent.com/849817/126190239-a602f8bb-755d-47ca-8a5a-f34429d7f363.jpeg)

If it did use the favicon as a fallback I would be worried the result would be something like below.  The favicon uses the full width / height due to the tight 16x16 favicon size.  For the apple touch icon we want some white space surrounding the symbol.
![Screen Shot 2021-07-19 at 11 59 38 AM](https://user-images.githubusercontent.com/849817/126190450-edca8436-16bb-461a-945f-9d109705047e.png)

Reply:
Ah ok, so this is just another Apple quirk.

On Android, it knows how to handle favicons
![image](https://user-images.githubusercontent.com/9084735/126191562-ed24d8ab-1333-4b44-a5e5-926038d8ffa6.png)